Tests/LibWeb/Layout/expected/table/percentage-width-columns.txt

```diff
@@ -1,33 +1,33 @@
 Viewport <#document> at (0,0) content-size 800x600 children: not-inline
   BlockContainer <html> at (0,0) content-size 800x600 [BFC] children: not-inline
-    BlockContainer <body> at (8,8) content-size 784x22.46875 children: not-inline
-      TableWrapper <(anonymous)> at (8,8) content-size 88.328125x22.46875 [BFC] children: not-inline
```

Comment:
`88.328125` -> `93.328125` is more difference then I would expect to see considering the changes. regression?